### PR TITLE
Feature/creating a succes modal message

### PR DESCRIPTION
**Pull request by moua0061**
## Description

This feature adds a modal success message when instructor successfully creates a new course after clicking the submit button. 

## Loom Video

https://www.loom.com/share/db3f2cb0969e4e4dad99d6b3d97e0bf3

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by MaryAngelique — APPROVED** [review]
The code quality is amazing though there is one warning which is a minor warning.

**Review by Raj-04 — APPROVED** [review]

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js [hunk 1]
```diff
@@ -88,11 +90,28 @@ const InstructorAddCourseCards = props => {
     schema.isValid(classData).then(valid => setDisabled(!valid));
   }, [classData]);
 
+  function success() {
+    Modal.success({
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:94]: Nice. Clear and concise

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js [hunk 2]
```diff
@@ -104,14 +123,23 @@ const InstructorAddCourseCards = props => {
           onChange={handleChange}
         >
           <Form.Item>
-            <label for="courseType">Course Type: </label>
+            <label for="courseType">Program Type: </label>
             <Select placeholder="Select a course" name="course_type">
-              {courses.map(course => (
+              {programs.map(course => (
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:128]:
```suggestion
              {programs.map(program=> (
```

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js [hunk 3]
```diff
@@ -104,14 +123,23 @@ const InstructorAddCourseCards = props => {
...
                 <Option value={course}>{course}</Option>
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:129]:
```suggestion
                <Option value={program}>{program}</Option>
```

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js (hunk 2 above)
Comment (jmerz826) [reply src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:128]: For consistency/readability (typ.)

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js [hunk 4]
```diff
@@ -229,5 +260,5 @@ const mapStateToProps = state => {
 };
 
 export default connect(mapStateToProps, { addProgram, setError })(
-  InstructorAddCourseCards
+  InstructorAddCourseForm
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:263]: much better naming! nice job

File: src/components/pages/InstructorAddCourse/InstructorAddedCoursesList.js [hunk 5]
```diff
@@ -8,17 +8,8 @@ const InstructorProgramList = props => {
...
-          marginBottom: 10,
-          fontSize: 20,
-          display: 'flex',
-          justifyContent: 'center',
-        }}
-      >
-        Already Submitted Programs:
-      </h1>
-      <List style={{ display: 'flex', justifyContent: 'center' }}>
+      <h1 className="instructor-h1">Already Submitted Programs:</h1>
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddedCoursesList.js:11]:
```suggestion
      <h1 className="instructor-h1">Current Programs:</h1>
```

Comment (jmerz826) [reply src/components/pages/InstructorAddCourse/InstructorAddedCoursesList.js:11]: Also, are we showing courses or programs?

File: src/components/pages/InstructorAddCourse/index.js [hunk 6]
```diff
@@ -1,26 +1,22 @@
...
-          <InstructorProgramList />
-          <InstructorAddCourseCards />
-        </Content>
-      </Layout>
-    </div>
+    <Layout>
+      <InstructorSidebar />
+      <Content>
+        <Banner />
+        {/* <InstructorAddedCoursesList /> */}
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/index.js:16]: delete

File: src/index.js [hunk 7]
```diff
@@ -35,13 +35,16 @@ import ParentBooking from './components/pages/ParentBooking';
...
+import NewsfeedPutModal from './components/pages/InstructorNewsFeed/NewsFeedPutModal';
 import ParentNewsFeed from './components/pages/ParentNewsFeed';
 import PaymentSuccess from './components/pages/ParentHome/PaymentSuccess';
 import Cart from './components/pages/ParentHome/Cart';
 import ParentFamilyHome from './components/pages/ParentFamily/ParentFamilyHome';
 import NavBar from './components/common/NavBar';
+import AllClasses from './components/pages/InstructorHome/AllClassesView';
+import Messages from './components/pages/Messages';
+import Classroom from './components/pages/Classroom';
+// import Newfeed from './components/pages/n';
```
Comment (jmerz826) [inline src/index.js:47]: delete commented out import

**Review by jmerz826 — CHANGES_REQUESTED** [review]
Modal message looks nice. These PRs are getting messy with so much previous code, though.

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js (hunk 1 above)
Comment (moua0061) [reply src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:94]: I agree. I've sent Paul a Slack message on how to fix this issue. I believe what we were doing by mistake was pulling each branch down and not realizing that it had cause a merging mess. I'm working on this with the other frontend engineers to untangle this mess.

**Review by ElchinNasirov — APPROVED** [review]

File: src/components/pages/InstructorAddCourse/InstructorAddedCoursesList.js (hunk 5 above)
Comment (moua0061) [reply src/components/pages/InstructorAddCourse/InstructorAddedCoursesList.js:11]: These renaming conventions are being done in a separate branch. It should all be courses except for the Program Type field in the form.

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js (hunk 3 above)
Comment (moua0061) [reply src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:129]: the renaming is being done in a different branch feature/styling-instructor-add-course

**Review by jmerz826 — APPROVED** [review]

**Pull request merged**